release-schema.json

```diff
@@ -196,6 +196,48 @@
           "minLength": 1
         }
       }
+    },
+    "ExclusionGrounds": {
+      "properties": {
+        "sources": {
+          "title": "Sources",
+          "description": "The sources where the exclusion grounds are defined.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Source"
+          },
+          "wholeListMerge": true,
+          "uniqueItems": true,
+          "minItems": 1
+        }
+      }
+    },
+    "SelectionCriteria": {
+      "properties": {
+        "sources": {
+          "title": "Sources",
+          "description": "The sources where the selection criteria are defined.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Source"
+          },
+          "wholeListMerge": true,
+          "uniqueItems": true,
+          "minItems": 1
+        }
+      }
+    },
+    "Source": {
```

Comment:
We don't typically `$ref` strings, which is why OCDS Kit incorrectly adds `wholeListMerge`.

I suggest not using the `$ref` and having the duplication.

Also, [until 2023](https://github.com/OpenDataServices/lib-cove/commit/8e97601ccc263da6671f5ae7cc08c4ff9c2b6fdc), ODS' JSON Schema extensions expected `codelist` and `openCodelist` to be at the level of the array, not at the level of array items. We haven't checked whether it's safe across OCDS tooling to put these on array items.